src/OpenTelemetry.Exporter.OpenTelemetryProtocol/README.md

```diff
@@ -72,6 +72,24 @@ values of the `PeriodicExportingMetricReaderOptions`
 | `OTEL_METRIC_EXPORT_INTERVAL` | `ExportIntervalMilliseconds`                    |
 | `OTEL_METRIC_EXPORT_TIMEOUT`  | `ExportTimeoutMilliseconds`                     |
 
+The following environment variables can be used to override the default
+values of the attribute limits
+(following the [OpenTelemetry specification](https://github.com/open-telemetry/opentelemetry-specification/blob/v1.15.0/specification/sdk-environment-variables.md#attribute-limits)).
+
+* `OTEL_ATTRIBUTE_VALUE_LENGTH_LIMIT`
```

Comment:
Why would these be documented here instead of the SDK doc (and probably link from here)?

Reply:
As these are supported only by this package.

Reply:
I just noticed https://github.com/open-telemetry/opentelemetry-dotnet/issues/3297?notification_referrer_id=NT_kwDOAE1THbIzNjcwOTgzNTI1OjUwNjc1NDk#issuecomment-1329543249. I will check tomorrow

Reply:
@pellared I lied 😅. I misremembered that we have not yet plumbed the limit config into the Jaeger and Zipkin exporters.